File: xfile_move_month.pl
#!/usr/bin/perl
use POSIX 'strftime';
$month=POSIX::strftime('%Y%m',localtime(time()));
for (grep { -f  && !/pl$/} (glob("*")))
{
	$to_dir1="/media/sdb1/_x_file/xfile_$month/";
	if( ! -d $to_dir1)
	{
		print "mv $_  _xfile_$month/ \n";
	}
	else
	{
		print "mv $_ $to_dir1\n";
	}
}

File: xfile_move_month.py
import time;
def get_cur_month():
    return time.strftime("_xfile_%Y_%m",time.localtime())

import glob
import os;
for file in glob.glob("*"):
    if os.path.isfile(file):
        print "mv    {0} {1} ".format(file, get_cur_month());
